sources/functions/deno

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Deno installer
+
+install_deno() {
```

Comment:
You don't need to install Deno if you're using the binary release, since it bundles a lightweight version of the Deno runtime.